services/web-app/components/framework-icon/framework-icon.js

```diff
@@ -41,7 +41,7 @@ FrameworkIcon.propTypes = {
   className: PropTypes.string,
   /**
    * Framework key from the asset schema. See:
-   * https://github.com/carbon-design-system/carbon-platform/blob/main/docs/resource-schemas.md.
+   * https://bit.ly/3BPZoTw
```

Comment:
Did you mean to change this? This seems to link to a google doc 😉

Reply:
hehe whoops!